v2/.eslintrc.json

```diff
@@ -0,0 +1,238 @@
+{
```

Comment:
is this file copy/pasted from v1? if yes maybe we could use a symlink here as well?